themes/src/main/resources/theme/base/login/login-otp.ftl

```diff
@@ -23,31 +25,28 @@
                 </div>
             </#if>
 
-            <div class="${properties.kcFormGroupClass!}">
-                <div class="${properties.kcLabelWrapperClass!}">
-                    <label for="otp" class="${properties.kcLabelClass!}">${msg("loginOtpOneTime")}</label>
-                </div>
+            <div class="${properties.kcFormGroupClass!} ${properties.kcXLgMarginBottom!}">
+                <label for="otp" class="${properties.kcLabelClass!}">${msg("loginOtpOneTime")}</label>
+                <div class="${properties.kcInputWrapperClass!}">
+                    <input id="otp" name="otp" autocomplete="off" type="text" class="${properties.kcInputClass!}"
+                        autofocus aria-invalid="<#if messagesPerField.existsError('totp')>true</#if>"/>
 
-            <div class="${properties.kcInputWrapperClass!}">
-                <input id="otp" name="otp" autocomplete="off" type="text" class="${properties.kcInputClass!}"
-                       autofocus aria-invalid="<#if messagesPerField.existsError('totp')>true</#if>"/>
-
-                <#if messagesPerField.existsError('totp')>
-                    <span id="input-error-otp-code" class="${properties.kcInputErrorMessageClass!}"
-                          aria-live="polite">
-                        ${kcSanitize(messagesPerField.get('totp'))?no_esc}
-                    </span>
-                </#if>
+                    <#if messagesPerField.existsError('totp')>
+                        <span id="input-error-otp-code" class="${properties.kcInputErrorMessageClass!}"
+                            aria-live="polite">
+                            ${kcSanitize(messagesPerField.get('totp'))?no_esc}
+                        </span>
+                    </#if>
+                </div>
             </div>
-        </div>
 
             <div class="${properties.kcFormGroupClass!}">
                 <div id="kc-form-options" class="${properties.kcFormOptionsClass!}">
                     <div class="${properties.kcFormOptionsWrapperClass!}">
                     </div>
                 </div>
 
-                <div id="kc-form-buttons" class="${properties.kcFormButtonsClass!}">
+                <div id="kc-otp-form-buttons" class="${properties.kcFormButtonsClass!}">
```

Comment:
@JonSzeto821 what was the reason for changing this id?

Reply:
@tyandor - `kc-form-buttons` is tied to a declaration in the login.css file that stylizes how this element renders onto the screen. It's currently used across multiple .ftl files. I didn't want to modify the declaration, so I updated it to this new id to keep some sort of context within the code here without pulling in the styles tied to the original id